drivers/video/gc2145.c

```diff
@@ -1166,6 +1177,9 @@ static int gc2145_init(const struct device *dev)
 /* Unique Instance */
 static const struct gc2145_config gc2145_cfg_0 = {
 	.i2c = I2C_DT_SPEC_INST_GET(0),
+#if DT_INST_NODE_HAS_PROP(0, pwdn_gpios)
+	.pwdn_gpio = GPIO_DT_SPEC_INST_GET(0, pwdn_gpios),
```

Comment:
If a developer needs a multi-GC2145 device can add a `cfg->pwdn_gpios`, and that would be a different pull request